<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Test ct_error_SUITE:no_compile result</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<a name="top"></a><pre>
=== Test case: <a href="ct_error_suite.src.html#no_compile-1">ct_error_SUITE:no_compile/1</a> (click for source code)

=== Config value:

    [{trace_level,50},
     {ct_opts,[]},
     {ct_node,ct@0dc5b4eee7ea},
     {path_dirs,["/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test"]},
     {watchdog,&lt;0.5656.0&gt;},
     {tc_logfile,"/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@0dc5b4eee7ea.2024-11-12_13.31.17/make_test_dir.common_test_test.logs/run.2024-11-12_13.31.20/ct_error_suite.no_compile.html"},
     {tc_group_properties,[]},
     {tc_group_path,[]},
     {data_dir,"/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/"},
     {priv_dir,"/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@0dc5b4eee7ea.2024-11-12_13.31.17/make_test_dir.common_test_test.logs/run.2024-11-12_13.31.20/log_private/"},
     {nodenames,[]}]

=== Current directory is "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@0dc5b4eee7ea.2024-11-12_13.31.17"

=== Started at 2024-11-12 13:36:33

<br />
See Common Test logs here:

<a href="file:///buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@0dc5b4eee7ea.2024-11-12_13.31.17/make_test_dir.common_test_test.logs/run.2024-11-12_13.31.20/log_private/all_runs.html">/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@0dc5b4eee7ea.2024-11-12_13.31.17/make_test_dir.common_test_test.logs/run.2024-11-12_13.31.20/log_private/all_runs.html</a>
<a href="file:///buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@0dc5b4eee7ea.2024-11-12_13.31.17/make_test_dir.common_test_test.logs/run.2024-11-12_13.31.20/log_private/index.html">/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@0dc5b4eee7ea.2024-11-12_13.31.17/make_test_dir.common_test_test.logs/run.2024-11-12_13.31.20/log_private/index.html</a>

Event receiver &lt;20980.760.0&gt; started!


[RUN #1] Calling ct:run_test([{logdir,
                                  "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@0dc5b4eee7ea.2024-11-12_13.31.17/make_test_dir.common_test_test.logs/run.2024-11-12_13.31.20/log_private/"},
                              {net_dir,[]},
                              {event_handler,
                                  {ct_test_support_eh,
                                      [{cbm,ct_test_support},
                                       {trace_level,50}]}},
                              {suite,
                                  ["/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/no_compile_SUITE"]}]) on ct@0dc5b4eee7ea


Common Test starting (cwd is /buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@0dc5b4eee7ea.2024-11-12_13.31.17)


Event Handler ct_test_support_eh started with [{cbm,ct_test_support},
                                               {trace_level,50}]


Common Test: Running make in test directories...

Recompile: no_compile_SUITE

no_compile_SUITE.erl:30:1: badly formed 'include'
%   30| suite() -&gt;
%     | ^


{error,make_failed}



Updating /buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@0dc5b4eee7ea.2024-11-12_13.31.17/make_test_dir.common_test_test.logs/run.2024-11-12_13.31.20/log_private/index.html ... 
done

Updating /buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@0dc5b4eee7ea.2024-11-12_13.31.17/make_test_dir.common_test_test.logs/run.2024-11-12_13.31.20/log_private/all_runs.html ... 
done


[RUN #1] Got return value {error,
                              {make_failed,
                                  ["/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/no_compile_SUITE"]}} after 306 ms

Saving start opts on ct@0dc5b4eee7ea: [{halt_with,
                                        {ct_test_support,ct_test_halt}},
                                       {logdir,
                                        "/buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@0dc5b4eee7ea.2024-11-12_13.31.17/make_test_dir.common_test_test.logs/run.2024-11-12_13.31.20/log_private/"},
                                       {net_dir,[]},
                                       {event_handler,
                                        {ct_test_support_eh,
                                         [{cbm,ct_test_support},
                                          {trace_level,50}]}},
                                       {suite,
                                        ["/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/no_compile_SUITE"]}]

[RUN #2] Calling ct_run:script_start() on ct@0dc5b4eee7ea


Common Test v1.27.3 starting (cwd is /buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@0dc5b4eee7ea.2024-11-12_13.31.17)


Event Handler ct_test_support_eh started with [{cbm,ct_test_support},
                                               {trace_level,50}]


Common Test: Running make in test directories...

Recompile: no_compile_SUITE

no_compile_SUITE.erl:30:1: badly formed 'include'
%   30| suite() -&gt;
%     | ^


{error,make_failed}



Updating /buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@0dc5b4eee7ea.2024-11-12_13.31.17/make_test_dir.common_test_test.logs/run.2024-11-12_13.31.20/log_private/index.html ... 
done

Updating /buildroot/otp/lib/common_test/make_test_dir/ct_logs/ct_run.test_server@0dc5b4eee7ea.2024-11-12_13.31.17/make_test_dir.common_test_test.logs/run.2024-11-12_13.31.20/log_private/all_runs.html ... 
done


Test run failed! Reason:
{make_failed,["/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/no_compile_SUITE"]}



[RUN #2] Got exit status value 2 after 1302 ms


</pre>
<div class="ct_error_notify"><pre><b>*** CT Error Notification 2024-11-12 13:36:35.462 ***</b><a id=e-37 class="link-to-entry" href="#e-37">&#x1f517;</a>
ct_error_SUITE:no_compile failed on line 146
Reason: {badmatch,{error,{make_failed,["/buildroot/otp/lib/common_te...}

<a href="#end">Full error description and stacktrace</a>
</pre></div>
<pre>
<a name="end"></a>


=== Ended at 2024-11-12 13:36:35
=== Location: [{ct_error_SUITE,no_compile,<a href="ct_error_suite.src.html#146">146</a>},
              {test_server,ts_tc,1794},
              {test_server,run_test_case_eval1,1303},
              {test_server,run_test_case_eval,1235}]
=== === Reason: no match of right hand side value 
                 {error,{make_failed,["/buildroot/otp/lib/common_test/make_test_dir/common_test_test/ct_error_SUITE_data/error/test/no_compile_SUITE"]}}
  in function  ct_error_SUITE:no_compile/1 (ct_error_SUITE.erl, line 146)
  in call from test_server:ts_tc/3 (test_server.erl, line 1794)
  in call from test_server:run_test_case_eval1/6 (test_server.erl, line 1303)
  in call from test_server:run_test_case_eval/9 (test_server.erl, line 1235)
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2024 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Tue Nov 12 2024 13:36:33<!--/date--><br />
</div>
</center>
</body>
</html>
